js/app.js

```diff
@@ -86,7 +66,8 @@ async function loadNextBlogBatch() {
             scrollToItem(btnLoadMore.dataset.first_item);
         }
     } else {
-        var errorMessage = getApiErrorMessage();
+        var errorMessage = getNoResultsMessage();
+        if (json.error) { errorMessage = getApiErrorMessage(); }
```

Comment:
if you rewrite like this:
```javascript
const errorMessage = json.error ? getApiErrorMessage() : getNoResultsMessage()
```
you will avoid evaluating the `getNoResultsMessage` unnecessarily